tasks/utilities/task_broad_terra_tools.wdl

```diff
@@ -290,8 +296,14 @@ task export_taxon_tables {
       "bbduk_docker": "~{bbduk_docker}",
       "r1_mean_q_raw": "~{r1_mean_q_raw}",
       "r2_mean_q_raw": "~{r2_mean_q_raw}",
+      "combined_mean_q_raw": "~{combined_mean_q_raw}",
+      "combined_mean_q_clean": "~{combined_mean_q_clean}",
       "r1_mean_q_clean": "~{r1_mean_q_clean}",
-      "r2_mean_q_clean": "~{r2_mean_q_clean}",
```

Comment:
Was line 294 deleted on accident?

Looks like `r2_mean_q_clean` is now missing from this WDL, both in the input block as well as the python bits defining the key:value pairs

Reply:
No it was deleted intentionally because TheiaProk_Illumina_PE no longer outputs the r1 or r2 mean_q_clean values, only the combined_mean_q_clean. However, the r1_mean_q_clean value still needs to be in this file because it is outputted by TheiaProk_Illumina_SE

Reply:
got it, thank you!

Reply:
That is also why the r1_mean_q_clean output is optional - it is only present in TheiaProk_Illumina_SE